#include <event.h> failed because include path is missing
patch should send upstream

diff --git a/ipc/chromium/moz.build b/ipc/chromium/moz.build
index 62acea0..13617dd 100644
--- a/ipc/chromium/moz.build
+++ b/ipc/chromium/moz.build
@@ -117,6 +117,7 @@ if os_solaris:
     ]
 
 CXXFLAGS += CONFIG["TK_CFLAGS"]
+CXXFLAGS += CONFIG['MOZ_LIBEVENT_CFLAGS']
 
 include("/ipc/chromium/chromium-config.mozbuild")
 
